scripts/tox_install_cmd.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
why need bash?

Reply:
we actually call the script via `bash`. `bash` has different syntax than `sh`, correcting the line to reflect the shell the script is written for.